### ISSUE TITLE
Instructions for migrating from onnx-tf to tf-onnx

### ISSUE THREAD

**Issue by jtanios** [onnx/onnx-tensorflow#536]
The project README states:

>We have joined force with Microsoft to co-develop ONNX Tensorflow frontend. For current onnx-tf frontend users, please migrate to use tf-onnx (https://github.com/onnx/tensorflow-onnx) where our code had been merged into.

Can we add more instructions on how to migrate to `tensorflow-onnx`? It looks like that package is doing TF -> ONNX only, not ONNX -> TF. What am I missing?

**Comment by fumihwh** [comment]
@jtanios Yes, onnx-tensorflow frontend(tf->onnx) has been merged into tensorflow-onnx.
And now, onnx-tensorflow only do onnx->tf, tensorflow-onnx do tf->onnx.

**Comment by maxx2s** [comment]
@fumihwh From your comment, it sounds like [onnx-tf](https://github.com/onnx/onnx-tensorflow) has been merged into [onnx-tf](https://github.com/onnx/tensorflow-onnx).

I'm looking in [tensorflow-onnx](https://github.com/onnx/tensorflow-onnx) but can't seem to find where the onnx-tf code is.

Is there an example of someone using tensorflow-onnx to convert from onnx to tf?
Thanks!

**Comment by chinhuang007** [comment]
@maxx2s Please note that https://github.com/onnx/tensorflow-onnx only converts tf to onnx. To convert from onnx to tf, still use https://github.com/onnx/onnx-tensorflow.

**Comment by maxx2s** [comment]
Sounds good, thanks!

**Comment by theotheo** [comment]
I am confused by this section of the documentation. I spent a couple hours to figure out how to use `tf-onnx` to convert from onnx... But this issue point me that is impossible. 

What the best way to understand it? Is there a plan to migrate this codebase to `onnx-tensorflow` repo? Maybe it should be best to remove this comment till to migrations? Or plan to have two separate repositories?...

Anyway, it seems useful to clarify this part somehow.
